src/main/resources/application.properties

```diff
@@ -4,5 +4,5 @@ spring.datasource.username=root
 spring.datasource.password=MySQL1234
```

Comment:
As I know, it's bad practice to pass the login and password to connect to the database in the configuration files.